doc/source/cluster/cloud.rst

```diff
@@ -37,13 +37,11 @@ AWS/GCP/Azure
 
             # Get a remote screen on the head node.
             $ ray attach ray/python/ray/autoscaler/aws/example-full.yaml
-            $ source activate tensorflow_p36
             $ # Try running a Ray program with 'ray.init(address="auto")'.
 
             # Tear down the cluster.
             $ ray down ray/python/ray/autoscaler/aws/example-full.yaml
 
-        .. tip:: For the AWS node configuration, you can set ``"ImageId: latest_dlami"`` to automatically use the newest `Deep Learning AMI <https://aws.amazon.com/machine-learning/amis/>`_ for your region. For example, ``head_node: {InstanceType: c5.xlarge, ImageId: latest_dlami}``.
```

Comment:
@richardliaw We can keep this if you think it is necessary. I'm inclined to remove it because
1. It is not quite accurate, we use a pinned version that is not updated all that often. 
2. To push people more towards docker.

Reply:
sure, sounds good